### PR TITLE
generate missing Shortcut* objects

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
They are necessary to build a Gtk.ShortcutsWindow

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Rebased, should be ready I guess:)

File: src/auto/shortcuts_shortcut.rs [hunk 1]
```diff
@@ -0,0 +1,1015 @@
...
+        unsafe {
+            gobject_sys::g_object_set_property(
+                self.as_ptr() as *mut gobject_sys::GObject,
+                b"icon-set\0".as_ptr() as *const _,
+                Value::from(&icon_set).to_glib_none().0,
+            );
+        }
+    }
+
+    //pub fn get_property_shortcut_type(&self) -> /*Ignored*/ShortcutType {
```
Comment (sdroege) [inline src/auto/shortcuts_shortcut.rs:157]: To generate this, only the enum has to be generated or not?

**Comment by sdroege** [comment]
Looks good to me otherwise

**Comment by bilelmoussaoui** [comment]
Rebased & generated `Gtk.ShortcutType` enum

**Comment by sdroege** [comment]
@GuillaumeGomez Green

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**